appveyor.yml

```diff
@@ -26,99 +26,90 @@ matrix:
-    - node --version
-    - npm --version
-
-    # core tests
-    - npm run pretest
-    - npm run tests-only
-
-    # resolver tests
-    - cd .\resolvers\webpack && npm test && cd ..\..
-    - cd .\resolvers\node && npm test && cd ..\..
 
-  on_success:
-    - npm run coveralls
--
-  matrix:
-    only:
-      - configuration: WSL
-  # Install scripts. (runs after repo cloning)
-  install:
-    # Get the latest stable version of Node.js or io.js
-    - ps: $env:WSLENV += ":nodejs_version"
-    - ps: wsl curl -sL 'https://deb.nodesource.com/setup_${nodejs_version}.x' `| sudo APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 -E bash -
-    - wsl sudo DEBIAN_FRONTEND=noninteractive apt install -y nodejs
-
-    # install modules
-    - ps: >-
-        if ($env:nodejs_version -eq "4") {
-          wsl sudo npm install -g npm@3;
-        }
-        if ($env:nodejs_version -in @("8", "10", "12")) {
-          wsl sudo npm install -g npm@6.10.3;
+build_script:
+  - ps: $true
+
+# Post-install test scripts.
+test_script:
+  # Output useful info for debugging.
+  - ps: Invoke-CICommand node --version
+  - ps: Invoke-CICommand npm --version
+
+  # core tests
+  - ps: Invoke-CICommand npm run pretest
+  - ps: Invoke-CICommand npm run tests-only
+
+  # resolver tests
+  - ps: >-
+      Get-ChildItem -Directory ./resolvers/ |
+        ForEach {
+          Push-Location $_;
+          Invoke-CICommand npm test;
+          Pop-Location;
         }
-    - wsl npm install
-
-    # fix symlinks
-    - git config core.symlinks true
-    - git reset --hard
-    # reset new-line characters
-    - wsl git reset --hard
-
-    # todo: learn how to do this for all .\resolvers\* on Windows
-    - cd .\resolvers\webpack && wsl npm install && cd ..\..
-    - cd .\resolvers\node && wsl npm install && cd ..\..
 
-  # Post-install test scripts.
-  test_script:
+on_success:
+  - ps: Invoke-CICommand npm run coveralls
 
-    # Output useful info for debugging.
-    - wsl node --version
-    - wsl npm --version
-
-    # core tests
-    - wsl npm run pretest
-    - wsl npm run tests-only
-
-    # resolver tests
-    - cd .\resolvers\webpack && wsl npm test && cd ..\..
-    - cd .\resolvers\node && wsl npm test && cd ..\..
-
-  on_success:
-    - wsl npm run coveralls
+# Configuration-specific steps
+for:
+  - matrix:
+      except:
+        - configuration: WSL
+    install:
+      # Get the latest stable version of Node.js or io.js
+      - ps: Install-Product node $env:nodejs_version
+  - matrix:
+      only:
+        - configuration: WSL
+    # Install scripts. (runs after repo cloning)
+    install:
+      # Get the latest stable version of Node.js
+      - ps: $env:WSLENV += ":nodejs_version"
+      - ps: wsl curl -sL 'https://deb.nodesource.com/setup_${nodejs_version}.x' `| sudo APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=1 -E bash -
+      - wsl sudo DEBIAN_FRONTEND=noninteractive apt install -y nodejs
 
 # Don't actually build.
-build: off
+build: on
```

Comment:
can you elaborate on this step?

Reply:
Yeah - before running `test_scripts` there are only five individual steps (`install`, `before_build`, `build_scripts` and `after_build` and `before_test`). Disabling builds takes us three steps short (the only individual steps would be `install` and `before_tests`)

I had to enable building because I need at least 3 steps:
1. Install nodejs (separate for `Native` and `WSL`
5. `npm install` (one common script for both `Native` and `WSL`
6. Upgrade nyc (separate script for `Native` as seen in my other PR)